shared/util.go

```diff
@@ -1076,7 +1076,6 @@ func RunCommandContext(ctx context.Context, name string, arg ...string) (string,
 
 // RunCommand runs a command with optional arguments and returns stdout. If the command fails to
 // start or returns a non-zero exit code then an error is returned containing the output of stderr.
-// Deprecated: Use RunCommandContext.
```

Comment:
Its still deprecated and places where its used should be updated.

Reply:
I think that's a lot of places were we'd just add a placeholder context, no?:
```
$ git grep -F 'shared.RunCommand(' | wc -l
254
```

Reply:
OK, but the function should still be marked as deprecated as we should avoid using it and guide people to using the other function that supports cancellation via context.

Reply:
Alright, I've properly marked it as deprecated which forced me to annotate a bunch of call sites with `// nolint: staticcheck` as I'm not sure how to build a proper context.

Also, is that cancellable context something that would allow me to `Ctrl-C` an operation I feel is too long? Like an image refresh for example?

Reply:
> Alright, I've properly marked it as deprecated which forced me to annotate a bunch of call sites with `// nolint: staticcheck` as I'm not sure how to build a proper context.
> 
> Also, is that cancellable context something that would allow me to `Ctrl-C` an operation I feel is too long? Like an image refresh for example?

You can pass context.TODO() and switch to non deprecated function if youve had to update old call sites anyway.